src/mlpack/methods/reinforcement_learning/environment/continuous_mountain_car.hpp

```diff
@@ -86,7 +87,35 @@ class ContinuousMountainCar
   {
     double action[1];
     // Storing degree of freedom
-    const int size = 1;
+    static constexpr int size = 1;
```

Comment:
Hm, I think that should work, but looking at the build error:

`rl_components_test.cpp:92: undefined reference to mlpack::rl::ContinuousMountainCar::Action::size'`

Can you just use `const int size = 1;` to see if that would fix the issue?